cmd/wio/commands/pac/user/login.go

```diff
@@ -0,0 +1,54 @@
+package user
+
+import (
+    "errors"
+    "wio/cmd/wio/commands"
+    "wio/cmd/wio/log"
+    "wio/cmd/wio/toolchain/npm/login"
+)
+
+type loginArgs struct {
+    dir   string
+    name  string
+    pass  string
+    email string
+}
+
+func (cmd Login) getArgs() (*loginArgs, error) {
+    ctx := cmd.Context
+    dir, err := commands.GetDirectory(cmd)
+    if err != nil {
+        return nil, err
+    }
+    args := ctx.Args()
+    if len(args) < 3 {
+        return nil, errors.New("wio login [username] [password] [email]")
```

Comment:
we should make password masked like *****. 

https://github.com/howeyc/gopass
https://stackoverflow.com/questions/2137357/getpasswd-functionality-in-go

Reply:
was too lazy to add command-line interface you can do it :)

Reply:
lol fine